docs/community/topics/i18n.rst

```diff
@@ -1,120 +1,112 @@
------------------------------------
+Adding or changing natural language text
+-----------------------------------------
 
-These steps cover how to add or change text that has been marked as translateable.
+If you add or change natural language text in the theme, you must update the message catalogs to include the new or updated text. Follow these steps:
 
-#. Edit the natural language text as desired.
-   Ensure that it is {ref}`marked as translateable <adding-natural-language-text>`.
+#. Edit the natural language text and ensure it is :ref:`marked as translatable <adding-translatable-text>`.
 
-#. Generate/update the message catalog template (``POT`` file) with `the PyBabel extract command <https://babel.pocoo.org/en/latest/cmdline.html#extract>`__:
+#. Generate/update the message catalog template (``POT`` file):
 
    .. code-block:: bash
+      
+      tox run -e i18n-extract
 
-      pybabel extract . -F babel.cfg -o src/pydata_sphinx_theme/locale/sphinx.pot -k '_ __ l_ lazy_gettext'
-
-   **To run this in ``.nox``**: ``nox -s translate -- extract``.
-
-#. Update the message catalogs (``PO`` files) with `the PyBabel update command <https://babel.pocoo.org/en/latest/cmdline.html#update>`__:
+#. Update the message catalogs (``PO`` files) for the existing locales:
 
    .. code-block:: bash
 
-      pybabel update -i src/pydata_sphinx_theme/locale/sphinx.pot -d src/pydata_sphinx_theme/locale -D sphinx
-
-   **To run this in ``.nox``**: ``nox -s translate -- update``.
+      tox run -e i18n-update
 
+This will update the translation files with new information about the position and text of the language you have modified.
 
-This will update these files with new information about the position and text of the language you have modified.
+If you *only* change non-translatable text (like HTML markup), the ``extract`` and ``update`` commands will only update the
+positions (line numbers) of the translatable strings.
+Updating positions is optional - the line numbers are to inform the human translator, not to perform the translation.
 
-If you *only* change non-translatable text (like HTML markup), the `extract` and `update` commands will only update the positions (line numbers) of the translatable strings. Updating positions is optional - the line numbers are to inform the human translator, not to perform the translation.
+If you change translatable strings, the ``extract`` command will extract the new or updated strings to the ``POT`` file,
+and the ``update`` command will try to perform a fuzzy match between the new or updated strings and existing translations in the ``PO`` files.
+If there is a fuzzy match, a comment like ``#, fuzzy`` is added before the matched entry,
+this means that the translation needs to be manually reviewed and possibly updated.
+If after reviewing the translation you decide to keep the existing translation, you can remove the ``#, fuzzy`` comment from the entry.
+Otherwise, it will add a new translation entry.
```

Comment:
the scope of this "otherwise" is a bit unclear to me. Does it mean "if there is a non-fuzzy match" (i.e., a perfect match)?  Or does it mean "if there is no match"?  I think it's the latter.  It might help to switch the order of information, like:

> If no match is found, a new translation entry will be automatically added.
> If a match *is* found, a comment is added to the matched entry...

Reply:
I mean if there is no fuzzy match - noting for rewording here